runner.yml.dist

```diff
@@ -20,6 +20,7 @@ drupal:
     - "./vendor/bin/drush config-set node.settings use_admin_theme 1 -y"
     # Enable the modules.
     - "./vendor/bin/drush en config_devel -y"
+    - "./vendor/bin/drush en oe_editorial -y"
```

Comment:
Why is this required?

Reply:
oe_editorial is not a requirement of oe_editorial_demo so it doesn't get installed by default. Do you think we should add it as a dependency?

Reply:
Yeah right, if you go through the chain, oe_editorial_demo->oe_editorial_entity_version->oe_editorial_corporate_workflow->oe_editorial, so please add oe_editorial as dependency to oe_editorial_corporate_workflow instead of having it in the runner.